test/Microsoft.Extensions.FileProviders.Physical.Tests/Microsoft.Extensions.FileProviders.Physical.Tests.csproj

```diff
@@ -1,17 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
-
```

Comment:
I rather like these new-lines: keeps the file organized in logical chucks. I realize VS 2017 deleted these automatically. Can you reverse this change?